lib/logstash/inputs/udp.rb

```diff
@@ -50,6 +51,7 @@ def run(output_queue)
       udp_listener(output_queue)
     rescue => e
       @logger.warn("UDP listener died", :exception => e, :backtrace => e.backtrace)
+      @metric_errors.increment(:listener_exception)
```

Comment:
I am worrying about the number of the similar name of keys in the plugins, should we define an `exception` namespace and define keys in it.

Maybe something like this:

```json
{
exceptions: {
listener: 123,
worker: 1234
}
}
```